lib/project/pro_motion_form/form_screen.rb

```diff
@@ -0,0 +1,8 @@
+module ProMotion
+  class FormScreen
+    # Defining this here is critical for PM::FormScreens to not fire delegate
+    # methods twice. This class must be defined even if we aren't using
+    # the ProMotion-form gem so that our pm_handles_delegates? method
+    # works properly for those people who _do_ use PM::FormScreen
+  end
```

Comment:
Awesome. Glad you liked the suggestion. The other way worked but this just felt more clear especially if we hit this again on another screen. 

Sent from my iPhone

> On May 6, 2015, at 5:19 PM, Mark Rickert <notifications@github.com> wrote:
> 
> In lib/project/pro_motion_form/form_screen.rb:
> 
> > @@ -0,0 +1,8 @@
> > +module ProMotion
> > +  class FormScreen
> > +    # Defining this here is critical for PM::FormScreens to not fire delegate
> > +    # methods twice. This class must be defined even if we aren't using
> > +    # the ProMotion-form gem so that our pm_handles_delegates? method
> > +    # works properly for those people who _do_ use PM::FormScreen
> > +  end
> Good call there. I've updated the PR to do this.
> 
> —
> Reply to this email directly or view it on GitHub.
>